modules/rhcos-enabling-multipath-day-2.adoc

```diff
@@ -10,7 +10,7 @@
 
 [IMPORTANT]
 ====
-Enabling multipathing during installation is supported and recommended for nodes provisioned in {product-title} 4.8 or higher. In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time. For more information about enabling multipathing during installation time, see "Enabling multipathing with kernel arguments on RHCOS" in the _Installing on bare metal_ documentation.
+Enabling multipathing during installation is supported and recommended for nodes provisioned in {product-title} 4.8 or higher. In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time. For more information about enabling multipathing during installation time, see "Enabling multipathing post installation" in the _Installing on bare metal_ documentation.
```

Comment:
Suggest moving this to L9 after the H1 heading.

Reply:
Thanks @Srivaralakshmi for the comments/suggestions. I have moved above note to line 9.